src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1163,6 +1165,8 @@ internal static DeclarationModifiers GetModifier(SyntaxKind kind, SyntaxKind con
                             return DeclarationModifiers.Partial;
                         case SyntaxKind.AsyncKeyword:
                             return DeclarationModifiers.Async;
+                        case SyntaxKind.ScopedKeyword:
+                            return AllowScopedModifier() ? DeclarationModifiers.Scoped : DeclarationModifiers.None;
```

Comment:
Wouldn't it be better if the keyword was parsed regardless of language version, but in a lower version it would show an error that the feature isn't available? That would make for a better experience when users try this feature but didn't upgrade their lang version, and would also allow the language version code fix to work.

Reply:
Thanks. I'd initially expected breaking changes parsing `scoped` as a modifier, so I'd made the modifier conditional on language version, but the syntax may be unambiguous after all. I've updated the parser to allow `scoped` as a modifier unconditionally, and moved the language version check to binding.